crates/core/app/tests/app_can_define_and_delegate_to_a_validator.rs

```diff
@@ -35,8 +35,11 @@ async fn app_can_define_and_delegate_to_a_validator() -> anyhow::Result<()> {
     let storage = TempStorage::new().await?;
 
     // Configure an AppState with slightly shorter epochs than usual.
-    let app_state =
-        AppState::Content(genesis::Content::default().with_epoch_duration(EPOCH_DURATION));
+    let app_state = AppState::Content(
+        genesis::Content::default()
+            .with_epoch_duration(EPOCH_DURATION)
+            .with_chain_id(TestNode::<()>::CHAIN_ID.to_string()),
```

Comment:
Just for clarification: was this necessary to make the change? It seems desirable to allow the chain ID to be `""` for testing, and it seems like there's no downside since any real network _will_ have a chain ID (which wherefore won't match).

Reply:
It's not necessary, and an empty chain id would work!